docs/api.rst

```diff
@@ -0,0 +1,59 @@
+Main operations of web app
+==========================
+
+Entry point
+-----------
+.. automodule:: webapp.app
+    :members:
+    :inherited-members:
+
+All web routes
+--------------
+.. automodule:: webapp.routes
+    :members:
+    :inherited-members:
+
+All socket channels
+-------------------
+.. automodule:: webapp.sockets
+    :members:
+    :inherited-members:
+
+Supporting modules
+==================
+
+Camera Manager
+--------------
+.. automodule:: webapp.inputs.camera_manager
+    :members:
+    :inherited-members:
+
+Virtual Terminal
+----------------
+.. automodule:: webapp.utils.virtual_terminal
+    :members:
+    :inherited-members:
+
+Fernet Vault
+------------
+.. automodule:: webapp.utils.file_encryption
+    :members:
+    :inherited-members:
+
+User Management
+===============
+.. automodule:: webapp.users
+    :members:
+    :inherited-members:
```

Comment:
I think maybe we can use the `:show-inheritence:` directive instead of `:inhertited-members:`. This suggestion will list the base class(s) as a hyperlinked reference instead of directly pulling documentation from other libraries (which is already confusing -- ie [the query class](https://vve-webapp.readthedocs.io/en/latest/api.html#webapp.users.User.query_class)).